flake8_eradicate.py

```diff
@@ -73,12 +75,32 @@ def parse_options(cls, options) -> None:
 
     def __iter__(self) -> Iterable[Tuple[int, str]]:
         """Runs on each step of flake8."""
-        if not self._is_equal_source():
+        if self._contains_commented_out_code():
             yield (1, self._error_template)
 
-    def _is_equal_source(self) -> bool:
-        filtered_source = ''.join(filter_commented_out_code(
-            self._physical_line, self._options,
-        ))
+    def _contains_commented_out_code(self) -> bool:
+        """
+        Check if the current physical line contains commented out code.
+
+        This test relies on eradicate function to remove commented out code
+        from a physical line.
 
-        return self._physical_line == filtered_source
+        Physical lines might appear like commented code although they are part
+        of a multi-line docstring (e.g. a `# noqa: DAR201` comment to suppress
+        flake8 warning about missing returns in the docstring).
+        To prevent this false-positive, the tokens of the physical line are
+        checked for a comment. The eradicate function is only invokes,
+        when the tokens indicate a comment in the physical line.
+
+        """
+        comment_in_line: bool = False
```

Comment:
you can remove the annotation here

Reply:
Thank you for the feedback! 

I have removed the annotation in https://github.com/sobolevn/flake8-eradicate/pull/106/commits/8ee93a42ecfb065e0daa22d9a8695a5650807e3e